xml/System.IO/BufferedStream.xml

```diff
@@ -1516,9 +1585,19 @@ bufStream->Close();
         <Parameter Name="buffer" Type="System.ReadOnlySpan&lt;System.Byte&gt;" Index="0" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="buffer">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="buffer">A region of memory. This method copies the contents of this region to the current buffered stream.</param>
+        <summary>When overridden in a derived class, writes a sequence of bytes to the current buffered stream and advances the current position within this buffered stream by the number of bytes written.</summary>
```

Comment:
If BufferedStream is sealed, Write can't be overridden.

Reply:
Oh, this was a mistake from me. I based this summary on Stream, but I did not remove that first part. It should be:

```suggestion
        <summary>Writes a sequence of bytes to the current buffered stream and advances the current position within this buffered stream by the number of bytes written.</summary>
```